main/templates/admin/bit/google_oauth.html

```diff
@@ -0,0 +1,11 @@
+{{
+  forms.panel_fields(
+      'Google',
+      (form.google_client_id, form.google_client_secret),
+      '''
+        Callback URL for <a href="https://console.developers.google.com/project/%s/apiui/credential">Google application</a>:
```

Comment:
It says `Redirect URI`.. I'm usually using exactly what is says there to find it easier:
![Uploading Screen Shot 2015-03-18 at 00.26.57.png . . .]()